packages/opencanary/data_stream/events/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,759 @@
+        "11001": "LOG_NTP_MONLIST"
+        "12001": "LOG_VNC"
+        "13001": "LOG_SNMP_CMD"
+        "14001": "LOG_RDP"
+        "15001": "LOG_SIP_REQUEST"
+        "16001": "LOG_GIT_CLONE_REQUEST"
+        "17001": "LOG_REDIS_COMMAND"
+        "18001": "LOG_TCP_BANNER_CONNECTION_MADE"
+        "18002": "LOG_TCP_BANNER_KEEP_ALIVE_CONNECTION_MADE"
+        "18003": "LOG_TCP_BANNER_KEEP_ALIVE_SECRET_RECEIVED"
+        "18004": "LOG_TCP_BANNER_KEEP_ALIVE_DATA_RECEIVED"
+        "18005": "LOG_TCP_BANNER_DATA_RECEIVED"
+        "99000": "LOG_USER_0"
+        "99001": "LOG_USER_1"
+        "99002": "LOG_USER_2"
+        "99003": "LOG_USER_3"
+        "99004": "LOG_USER_4"
+        "99005": "LOG_USER_5"
+        "99006": "LOG_USER_6"
+        "99007": "LOG_USER_7"
+        "99008": "LOG_USER_8"
+        "99009": "LOG_USER_9"
+      source: >
+        def logType = ctx.opencanary.logtype;
+        if (ctx.log == null) {
+          ctx.log = new HashMap();
+        }
+        if (params.get(logType.toString()) == null) {
+          ctx.log['logger'] = logType;
+        } else {
+          ctx.log['logger'] = params.get(logType.toString());
+        }
+        ctx.opencanary.remove('logtype');
+  - rename:
+      description: Rename node_id to opencanary.node.id
+      tag: Rename node_id to opencanary.node.id
+      field: opencanary.node_id
+      target_field: opencanary.node.id
+      ignore_missing: true
+      ignore_failure: true
+  - rename:
+      description: Rename SKIN to opencanary.skin
+      tag: Rename SKIN to opencanary.skin
+      field: opencanary.logdata.SKIN
+      target_field: opencanary.skin
+      ignore_missing: true
+      ignore_failure: true
+
+  # Convert some field types
+  - convert:
+      description: Convert opencanary.logdata.ID to long
+      tag: Convert opencanary.logdata.ID to long
+      field: opencanary.logdata.ID
+      type: long
+      ignore_missing: true
+      ignore_failure: true
+  - convert:
+      description: Convert opencanary.logdata.TTL to long
+      tag: Convert opencanary.logdata.TTL to long
+      field: opencanary.logdata.TTL
+      type: long
+      ignore_missing: true
+      ignore_failure: true
+  - convert:
+      description: Convert opencanary.logdata.URGP to long
+      tag: Convert opencanary.logdata.URGP to long
+      field: opencanary.logdata.URGP
+      type: long
+      ignore_missing: true
+      ignore_failure: true
+  - convert:
+      description: Convert opencanary.logdata.WINDOW to long
+      tag: Convert opencanary.logdata.WINDOW to long
+      field: opencanary.logdata.WINDOW
+      type: long
+      ignore_missing: true
+      ignore_failure: true
+      
+
+  # MSSQL Events
+  - rename:
+      description: Rename MSSQL event opencanary.logdata.AppName to opencanary.mssql.client.app fields.
+      tag: Rename MSSQL event opencanary.logdata.AppName to opencanary.mssql.client.app fields.
+      if: ctx.log?.logger != null && ctx.log.logger == 'LOG_MSSQL_LOGIN_SQLAUTH'
```

```suggestion
      if: ctx.log?.logger == 'LOG_MSSQL_LOGIN_SQLAUTH'
```
If the field's value equals the string, it cannot have equaled null, so the null check is not required. Same below.